docs/advanced-usage/access-control/private-docker-registries/index.md

```diff
@@ -0,0 +1,9 @@
+# Private Docker registries
+
+If your workflow needs to access private Docker images follow the docs depending on the Docker registry.
+
+## CERN GitLab
+
+Go to your project's member tab (`https://gitlab.cern.ch/<you-username>/<your-repo-name>/-/project_members`) and add the [REANA user](https://gitlab.cern.ch/reana) as a collaborator:
```

Comment:
I see in the screenshot that the REANA user is added with role "Reporter". Assuming this is needed, should we mention it? I see "as a collaborator" but it doesn't specify which role.